torch/csrc/autograd/python_hook.cpp

```diff
@@ -157,8 +157,8 @@ static void check_single_result(PyObject* _original, PyObject* _result, PyObject
     throw python_error();
   }
 
-  auto& original = ((THPVariable*)_original)->cdata.data();
-  auto& result = ((THPVariable*)_result)->cdata.data();
+  auto& original = ((THPVariable*)_original)->cdata;
+  auto& result = ((THPVariable*)_result)->cdata;
```

Comment:
It's okay to remove `.data()` in these two statements, because the rest of the function does not perform any gradient-accumulating operations on `original` or `result`.